SPECS/nbdkit/nbdkit.spec

```diff
@@ -958,48 +894,440 @@ make %{?_smp_mflags} check
 %{_mandir}/man1/nbdkit-release-notes-1.*.1*
 %{_libdir}/pkgconfig/nbdkit.pc
 
+%if !0%{?exclude_pkg}
+%files srpm-macros
+%license LICENSE
+%{_rpmconfigdir}/fileattrs/nbdkit.attr
+%{_rpmconfigdir}/nbdkit-find-provides
+%endif
 
 %files bash-completion
 %license LICENSE
 %dir %{_datadir}/bash-completion/completions
 %{_datadir}/bash-completion/completions/nbdkit
 
-
 %changelog
-* Thu Apr 21 2022 Muhammad Falak <mwani@microsoft.com> - 1.20.7-5
-- Avoid non-zero exit from `%check` section
+* Mon Oct 31 2022 Ameya Usgaonkar <ausgaonkar@microsoft.com> - 1.30.10-3
```

Comment:
nit: changelog seems fishy

Reply:
The nbdkit package in EXTENDED spec directory is an old one (1.20) version.  It had refences to old libraries, such as libssh-devel that are no longer maintained.  Hence, I brought in latest release (1.30) from Fedora 36.